Fulfillment/Operation/01_ManageDispatchNotes.md

```diff
@@ -0,0 +1,140 @@
+
+6. Check the shipment status in the *Shipment status* column. For further information, see [Check the shipment status](./02_CheckShipment.md#check-the-shipment-status).
+
+7. If desired, click the ![Columns](../../Assets/Icons/Columns.png "[Columns]") (Columns) button to add further details to the list of dispatch notes. 
+    > [Info] Additional information can be displayed depending on the fields (attributes) mapped via the fulfiller's driver when establishing the connection. 
+
+
+## Complete a dispatch note
+
+Fulfillers may not notify if an order has been delivered. In this case, the dispatch note status is not automatically updated to completed. To provide a better overview and handling of the dispatch note list, the dispatch note status can be updated manually if necessary. It is also possible to update the status of several dispatch notes at once.
+
+#### Prerequisites
+
+- The shipment has been delivered.
+- The dispatch note status is **Partner confirmed receival** and the shipment status remains as ![Circular arrows](../../Assets/Icons/CircularArrows.png "[Circular arrows]") (Circular arrows), that is, in progress, in the list of dispatch notes. 
+- All shipments included in the dispatch note have been shipped, that is, the status of all shipments included in the *Shipments* sub-tab of the *Dispatch note "Dispatch note ID"* view is **Shipped**.
+
+#### Procedure
+
+*Fulfillment > Dispatch notes > Tab LIST*
+
+![Dispatch notes](../../Assets/Screenshots/Fulfillment/DispatchNotes/DispatchNotes.png "[Dispatch notes]")
+
+1. Click the dispatch note whose status has not been updated.  
+    The *Dispatch note "Dispatch note ID"* view is displayed.
+
+2. Click the *Shipments* sub-tab.  
+    All shipments included in the dispatch note are displayed.
+
+3. Verify that the status of all shipments is **Shipped**.
+
+4. Click the ![Back](../../Assets/Icons/Back02.png "[Back]") (Back) button to return to the list of dispatch notes. 
+
+5. Click the checkbox of the dispatch note(s) to be completed.   
+    The editing toolbar is displayed.
+
+6. Click the [FORCE COMPLETION] button in the editing toolbar.  
+    The *"x DispatchNotes forced to completion, failed for x"* notice is displayed.
+
+    > [Info] The x indicates the number of dispatch notes that have been completed or that have failed. A manual completion will only fail if the status of any shipment contained in the dispatch note is not **Shipped**.
+
+    ![Forced to completion](../../Assets/Screenshots/Fulfillment/DispatchNotes/ForcedtoCompletion.png "[Forced to completion]")
+
+    The *Status* column changes to **Dispatch note completed** and the *Shipment status* column changes to ![Double check](../../Assets/Icons/DoubleCheck.png "[Double check]") (Double check). The status of all shipments included in the dispatch note is updated to **Delivered**.    
```

Comment:
```suggestion
    The dispatch note status in the *Status* column changes to **Dispatch note completed** and the shipment status in the *Shipment status* column changes to ![Double check](../../Assets/Icons/DoubleCheck.png "[Double check]") (Double check). The status of all shipments in the dispatch note is updated to **Delivered**.    
```
Checken ob ich die Status richtig benannt habe. Aber wichtig ist, dass sich der Status in der Spalte und nicht die Spalte selbst (oder der Spaltenname) ändert.